technical-reference/reference-technical-articles/technical-doc-token-converter.md

```diff
@@ -0,0 +1,226 @@
+        - `amountOutMantissa`: 100 * 10^18. That is **100 USDC**
+
+### Get amount in
+
+View function `getAmountIn(uint256 amountOutMantissa, address tokenAddressIn, address tokenAddressOut) returns (uint256 amountInMantissa, uint256 amountConvertedMantissa)`, to request the amount of `tokenAddressIn` tokens that a sender should send to the contract, to receive `amountOutMantissa` tokens of `tokenAddressOut`, where:
+
+- Parameters:
+    - `amountOutMantissa`: the amount of `tokenAddressOut` tokens the sender would like to receive in the conversion. It is defined in terms of the mantissa of `tokenAddressOut`
+    - `tokenAddressIn`: the address of the token provided by the sender to get tokens of `tokenAddressOut`
+    - `tokenAddressOut`: the address of the token to get after the conversion
+- Returned values:
+    - `amountInMantissa`: the amount of `tokenAddressIn` tokens that the sender has to send to the contract to receive `amountConvertedMantissa` tokens of `tokenAddressOut`
+    - `amountConvertedMantissa`: the amount of `tokenAddressOut` tokens that would be really transferred from the contract to the sender if the user sends `amountInMantissa` tokens of `tokenAddressIn` to the contract. If there is enough liquidity of `tokenAddressOut` in the contract, this amount is equal to `amountOutMantissa`. Otherwise, it will be less than the amount
+
+Examples:
+
+- Invocation 1: `getAmountIn(40 * 10^18, address(XVS), address(USDC))`
+    - The configuration enables for `tokenAddressIn / tokenAddressOut`: XVS/USDC
+    - Current liquidity: 100 USDC
+    - Current conversion rate, given by the oracle: 1 XVS = 5 USDC
+    - Configured discount: 0%
+    - Response:
+        - `amountInMantissa`: 8 * 10^18 (XVS)
+        - `amountConvertedMantissa`: 40 * 10^18 (USDC), equal to the `amountOutMantissa` param
+- Invocation 2: `getAmountIn(120 * 10^18, address(XVS), address(USDC))`
+    - The configuration enables for `tokenAddressIn / tokenAddressOut`: XVS/USDC
+    - Current liquidity: 100 USDC
+    - Current conversion rate, given by the oracle: 1 XVS = 5 USDC
+    - Configured discount: 0%
+    - Response:
+        - `amountInMantissa`: 20 * 10^18 (XVS)
+        - `amountConvertedMantissa`: 100 * 10^18 (USDC), different from the `amountOutMantissa` param because there is not enough liquidity
+
+### Conversions fixing the amount in
+
+Function `convertExactTokens(uint256 amountInMantissa, uint256 amountOutMinMantissa, address tokenAddressIn, address tokenAddressOut, address to)`, where:
+
+- `amountInMantissa`: the number of tokens (`tokensAddressIn`) the sender wants to convert
+    - It’s defined in the mantissa of the `tokenAddressIn`
+    - Examples:
+        - 1 [TRX](https://bscscan.com/address/0xce7de646e7208a4ef112cb6ed5038fa6cc6b12e3) (6 decimals in BSC), will be 1000000
+        - 1 [USDC](https://bscscan.com/address/0x8ac76a51cc950d9822d68b83fe1ad97b32cd580d) (18 decimals in BSC), will be 1000000000000000000
```

Comment:
```suggestion
```
nit: I'm not sure this is necessary at this point. We have almost gotten to the end of the document and this is the first time we are explaining and giving examples what it means to define a token in its mantissa. I think it is out of scope